lib/mollie/method.rb

```diff
@@ -41,5 +42,16 @@ def normal_image
     def bigger_image
       image['size2x']
     end
+
+    def pricing=(pricing)
+      @pricing = pricing.map do |price|
+        OpenStruct.new(
+          description: price['description'],
+          fixed: Mollie::Amount.new(price['fixed']),
+          variable: price['variable'],
+          fee_region: price['fee_region']
```

Comment:
Shouldn't this be `feeRegion`?

```suggestion
          fee_region: price['feeRegion']
```

Reply:
The key names have already been transformed into snake case at this point:
https://github.com/mollie/mollie-api-ruby/blob/d885c5721a5b48e7cf59665a5dd913462b091d8c/lib/mollie/client.rb#L123